src/main/java/dukeApp/files/Deadline.java

```diff
@@ -0,0 +1,94 @@
+package dukeApp.files;
+
+public class Deadline extends Task {
+    protected String word;
+    protected String date;
+    protected String time;
+    protected String dateLine;
```

Comment:
What is the difference between dateLine and originalDate? Do you need so many variables with that share duplicated date & time?